classy_bot/__main__.py

```diff
@@ -68,7 +70,7 @@ async def imagine(
                 negative_prompt=negative_prompt,
                 width=1216,
                 height=832,
-                num_inference_steps=inference_steps
+                num_inference_steps=inference_steps # type: ignore
```

Comment:
why the `# type: ignore` ?
Consider either fixing the types or adding a comment explaining why it's ignored